README.md

```diff
@@ -0,0 +1,150 @@
+# [Arbitrum Token Bridge](https://bridge.arbitrum.io/)
+
+### The canonical token-bridge of [Arbitrum](https://arbitrum.io/)
+
+Are you a developer looking to explore the Arbitrum token bridge and its underlying technology? Then you've come to the right place! We'll walk you through the steps to run the Arbitrum token bridge website locally on your machine.
+
+<br />
+
+Arbitrum ❤️ Open-source
+
+Interested in contributing to this repo? We welcome your contribution.
+[Check out the contribution guidelines and instructions here](CONTRIBUTING.md).
+
+<br />
+
+---
+
+<br />
+
+## Prerequisites for running the code
+
+- Install [Node.js](https://nodejs.org/en/download/) for your platform
+- Install [Node Version Manager (nvm)](https://www.freecodecamp.org/news/node-version-manager-nvm-install-guide/) to setup the correct Node version for the project.
+- Install [VS Code](https://code.visualstudio.com/download) for your platform
+- Install [Yarn](https://classic.yarnpkg.com) - Package manager
```

Comment:
should we explicitly say v1? on the site there is a CTA for v2

Reply:
Unless we know for sure that yarn v2 works, we should explicitly say v1 IMO, we don't want contributors run into errors because they used yarn v2

Reply:
Done